frontend/src/index.js

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
 import { render } from 'react-dom';
 
-import App from './components/App';
+import MVP from './components/MVP';
```

Comment:
It was renamed because I just created a new component called `MVP` while `App` still existed, just so I could have them side by side while I switched it out. But I should change it to `App`, I think.

I've also given to some thought about maintaining the MVP branch just as a branch, and I've reversed course. I will merge this into master, and we can iterate out development from this point. If we want to use any of the old work, we can look into an older branch for the source.

Reply:
That makes sense. It might be useful to tag the current master branch before merging as the pre-MVP version so it can be referred back to easily.

Reply:
Whoops, too late haha